mgmtfn/pslibnet/main.go

```diff
@@ -67,7 +68,6 @@ var gcliOpts cliOpts
 func main() {
 	var flagSet *flag.FlagSet
 
-	log.SetFlags(log.LstdFlags | log.Lshortfile)
```

Comment:
I found `setFlag()` useful for easily modifying the log format (Example, having file name and line number comes in handy for debugging some failure without having to grep through code). 

I think logrus doesn't add this info by default and we could potentially define a custom formatter, if we want that info in logs later?